applets/notification_area/status-notifier/sn-item-v0.c

```diff
@@ -1395,7 +1413,7 @@ sn_item_v0_init (SnItemV0 *v0)
   v0->icon_size = 16;
   v0->effective_icon_size = 0;
   v0->image = gtk_image_new ();
-  gtk_container_add (GTK_CONTAINER (v0), v0->image);
+ gtk_button_set_image (GTK_BUTTON (v0), v0->image);
```

Comment:
This change is not related to the commit title.

Reply:
@rbuj 
If not changed, the status icon will not be displayed correctly after restarting the panel